cirq/ops/common_gates_test.py

```diff
@@ -620,13 +620,12 @@ def test_iswap_str():
 
 
 def test_iswap_unitary():
```

Comment:
I'm seeing different answers on https://quantumcomputing.stackexchange.com/questions/2594/what-is-the-matrix-of-the-iswap-gate. Some people say it should be `+i` while some people argue that it doesn't matter if it's `+i` or `-i`. Could someone please clear this up for me?

Reply:
There are two conflicting conventions. I think we have decided to use `-i` and already switched fSim in #1726. However, ISwap was missed in that PR and so @dabacon filed #1882. This PR fixes the inconsistency and completes our switch to the `-i` convention for both gates.